web-components/src/[sandbox]/sandbox.ts

```diff
@@ -118,6 +119,12 @@ export class Sandbox extends LitElement {
       <md-theme class="theme-toggle" id="app-theme" ?darkTheme=${this.darkTheme} ?lumos=${this.lumos}>
         ${this.themeToggle()}
         <elix-list-explorer class="explorer">
+          <div class="container" aria-label="md-dropdown">
+            <h2>md-dropdown</h2>
+            <sass-stats component="dropdown">
+              ${dropdownTemplate}
+            </sass-stats>
+          </div>
```

Comment:
Try to add this in an alphabetical order within this file.

Reply:
Done